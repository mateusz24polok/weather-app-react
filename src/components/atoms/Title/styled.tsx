import styled from "styled-components";

export const StyledTitle = styled.h1`
    font-size: 36px;
    font-weight: 700;
    text-align: center;
`