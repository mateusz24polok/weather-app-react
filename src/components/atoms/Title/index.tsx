import React from "react";
import { StyledTitle } from "./styled";

interface TitleProps {
  title: string;
}

const Title: React.FC<TitleProps> = ({ title }) => {
  return <StyledTitle>{title}</StyledTitle>;
};

export default Title;
