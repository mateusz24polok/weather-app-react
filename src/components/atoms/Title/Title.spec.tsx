import React from "react";
import { shallow } from "enzyme";
import Title from ".";

describe("<Title />", () => {
  it("should match snapshot", () => {
    const actual = <Title title="The Weather App" />;
    expect(actual).toMatchSnapshot();
  });
});

describe("render a text", () => {
  it("should render app title", () => {
    const wrapper = shallow(<Title title="The Weather App"/>);
    const text = wrapper.text();
    expect(text).toBe("The Weather App");

  });
})


