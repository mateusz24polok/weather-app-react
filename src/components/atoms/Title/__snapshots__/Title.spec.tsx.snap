// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should match snapshot 1`] = `
<Title
  title="The Weather App"
/>
`;
